.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--  Thanks for sending a pull request! -->
+
+**Is this a BUG FIX or a FEATURE ?**:
+
+> Uncomment only one, leave it on its own line:
+>
+> /kind bug
```

Comment:
/kind bot is not enabled on the repo. Unless we enable it, it should not be in the template.